src/scripts/make-webpack-config.ts

```diff
@@ -22,11 +22,10 @@ interface AliasedConfiguration extends Configuration {
 	resolve: Resolve & { alias: Record<string, string> };
 }
 
-export default function (
-	config: Rsg.SanitizedStyleguidistConfig,
-	env: 'development' | 'production' | 'none'
-): Configuration {
-	process.env.NODE_ENV = process.env.NODE_ENV || env;
+type EnvType = 'development' | 'production' | 'none';
```

Comment:
This type is available in webpack:

```suggestion
type Mode = Configuration['mode'];
```

https://github.com/styleguidist/react-styleguidist/blob/04d900d7271761c257eacf57bb5e7279ded3525a/src/scripts/make-webpack-config.ts#L16